rest/route/admin_restart.go

```diff
@@ -65,16 +75,17 @@ func (h *restartHandler) Run(ctx context.Context) gimlet.Responder {
 		EndTime:            h.EndTime,
 		User:               u.Username(),
 	}
-	resp, err := h.sc.RestartFailedTasks(h.queue, opts)
-	if err != nil {
-		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "Error restarting tasks"))
+	if h.restartType == evergreen.RestartVersions {
```

Comment:
I'm confused by the RestartVersions option only restarting commit queue versions.

Reply:
The purpose of this ticket is to restart failed commit queue items specifically. I imagine for non-commit queue admins could just restart failed tasks rather than whole versions.